src/main/java/mate/jdbc/dao/CarDaoImpl.java

```diff
@@ -0,0 +1,214 @@
+                + "JOIN cars c ON cd.car_id = c.id JOIN manufacturers m "
+                + "ON c.manufacturer_id = m.id WHERE cd.driver_id = ? AND c.is_deleted = FALSE";
+        List<Car> carList = new ArrayList<>();
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement statement = connection.prepareStatement(query)) {
+            statement.setLong(1, driverId);
+            ResultSet resultSet = statement.executeQuery();
+            while (resultSet.next()) {
+                carList.add(getCar(resultSet));
+            }
+        } catch (SQLException e) {
+            throw new RuntimeException("Can't get a list of cars from cars table. by driver_id "
+                    + driverId, e);
+        }
+        carList.forEach(car -> car.setDrivers(getDriversForCar(car.getId())));
+        return carList;
+    }
+
+    private void addDriverToCar(Car car) {
+        String query = "INSERT INTO cars_drivers (car_id, driver_id) VALUE (?, ?);";
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement statement = connection.prepareStatement(query)) {
+            statement.setLong(1, car.getId());
+            for (mate.jdbc.model.Driver driver : car.getDrivers()) {
+                statement.setLong(2, driver.getId());
+                statement.executeUpdate();
+            }
+        } catch (SQLException e) {
+            throw new RuntimeException("Can't insert drivers to car " + car, e);
+        }
+    }
+
+    private void deleteDriverFromCar(Car car) {
+        String query = "DELETE FROM cars_drivers WHERE car_id = ?";
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement statement = connection.prepareStatement(query)) {
+            statement.setLong(1, car.getId());
+            statement.executeUpdate();
+        } catch (SQLException e) {
+            throw new RuntimeException("Can't delete drivers of car " + car, e);
+        }
+    }
+
+    private List<Driver> getDriversForCar(Long carId) {
+        String getAllDriversForCarQuery = "SELECT id, name, license_number "
+                + "FROM drivers d JOIN cars_drivers cd "
+                + "ON d.id = cd.driver_id WHERE cd.driver_id = ?";
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement statement =
+                        connection.prepareStatement(getAllDriversForCarQuery)) {
+            statement.setLong(1, carId);
+            ResultSet resultSet = statement.executeQuery();
+            List<Driver> drivers = new ArrayList<>();
+            while (resultSet.next()) {
+                drivers.add(getDriverFromResultSet(resultSet));
+            }
+            return drivers;
+        } catch (SQLException e) {
+            throw new RuntimeException("Can't find driver for car in DB ",e);
+        }
+    }
+
+    private void insertDrivers(Car car) {
```

Comment:
Duplicated method. You already have the method `addDriverToCar` with the same logic.